### PR TITLE
Issue 233: Changes on EMT STATCOM model

### PR DESCRIPTION
**Pull request by Carlos-Alegre**
Pull request for the changes in the EMT STATCOM model.

### CODE REVIEW

File: pages/models/CompensationDevices/STATCOM/index.md [hunk 1]
```diff
@@ -101,6 +101,36 @@ There are two main control designs:
 
 In both cases, the DC voltage loop must be slower than the internal current control loop to ensure a stable response.
 
+## Current limitation
+
+You can visit the [Grid Following page](../../generations/Sources/VSC/EMTGridFollowingVSC/), which has a dedicated section on this topic.
+
+## Parameter tunning 
+
+The following table shows possible values for the parameters of the controllers using the tunning proposed:
```
Comment (Lampros-Cresym) [inline pages/models/CompensationDevices/STATCOM/index.md:110]:
```suggestion
The following table shows possible values for the parameters of the controllers using the tuning proposed:
```

File: pages/models/CompensationDevices/STATCOM/index.md [hunk 2]
```diff
@@ -101,6 +101,36 @@ There are two main control designs:
...
+
+| Parameter | Value | Units |
+| --------- | ----- | ----- |
+| $$V_{peak}$$ | $$2500$$ | V |
```
Comment (Lampros-Cresym) [inline pages/models/CompensationDevices/STATCOM/index.md:114]: please check if Vpeak appears anywhere.

File: pages/models/CompensationDevices/STATCOM/index.md [hunk 3]
```diff
@@ -101,6 +101,36 @@ There are two main control designs:
 
 In both cases, the DC voltage loop must be slower than the internal current control loop to ensure a stable response.
 
+## Current limitation
+
+You can visit the [Grid Following page](../../generations/Sources/VSC/EMTGridFollowingVSC/), which has a dedicated section on this topic.
+
+## Parameter tunning
```
Comment (Lampros-Cresym) [inline pages/models/CompensationDevices/STATCOM/index.md:108]:
```suggestion
## Parameter tuning 
```

**Review by Lampros-Cresym — APPROVED** [review]
minor changes.

File: pages/models/CompensationDevices/STATCOM/index.md (hunk 2 above)
Comment (Carlos-Alegre) [reply pages/models/CompensationDevices/STATCOM/index.md]: Chenged to Vdc

**Review by GJCRESYM — APPROVED** [review]

**Review by Lampros-Cresym — APPROVED** [review]

**Review by Lampros-Cresym — CHANGES_REQUESTED** [review]
Figures not appearing correctly. Probably a path problem?

**Comment by Carlos-Alegre** [comment]
> Figures not appearing correctly. Probably a path problem?

Hmmm I can see all of the figures. Can you post a screenshot?

**Review by Lampros-Cresym — APPROVED** [review]

**Pull request merged**